pkg/labeler.go

```
@@ -44,7 +44,8 @@ func NewTitleCondition() Condition {
 				return false, fmt.Errorf("title is not set in config")
 			}
 			log.Printf("Matching `%s` against: `%s`", matcher.Title, pr.GetTitle())
-			isMatched, _ := regexp.Match(matcher.Title, []byte(pr.GetTitle()))
+			re := regexp2.MustCompile(matcher.Title, 0)
+			isMatched, _ := re.MatchString(pr.GetTitle())
```

Never ignore errors :) 

Return both and let the caller decide what to do
```go
isMatched, err := re.MatchString(pr.GetTitle())
return isMatched, err
```